src/Scripting/CSharpTest.Desktop/CsiTests.cs

```diff
@@ -50,19 +51,21 @@ public void CurrentWorkingDirectory_Change()
 Environment.Exit(0)
 ");
 
-            AssertEx.AssertEqualToleratingWhitespaceDifferences($@"
-Microsoft (R) Visual C# Interactive Compiler version {s_compilerVersion}
-Copyright (C) Microsoft Corporation. All rights reserved.
+            var expected = $@"
+{ string.Format(CSharpScriptingResources.LogoLine1, s_compilerVersion) }
+{CSharpScriptingResources.LogoLine2}
 
-Type ""#help"" for more information.
+{ScriptingResources.HelpPrompt}
 > > > > > > 1
 > C {{ }}
 > 
-", result.Output);
+";
+            expected = expected.Replace((char)0x2013, (char)0x002d); // EN DASH -> HYPHEN-MINUS
```

Comment:
csi.exe outputs the [*en dash*](http://www.fileformat.info/info/unicode/char/2013/index.htm) in `Microsoft (R) Visual C# - interaktive Compilerversion 42.42.42.42424` as [*hyphen-minus*](http://www.fileformat.info/info/unicode/char/2d/index.htm). There might be a more reliable way to fix this.

Reply:
@MaStr11 Please add a more detailed comment explaining why the replacement is necessary. Something like:

> The German translation (and possibly others) contains an en dash (0x2013), but csi.exe outputs it as a hyphen-minus (0x002d). We need to fix up the expected string before we can compare it to the actual output.